server/pulp/server/managers/consumer/agent.py

```diff
@@ -10,7 +10,8 @@
 from pulp.plugins.loader import api as plugin_api, exceptions as plugin_exceptions
 from pulp.plugins.model import Consumer as ProfiledConsumer
 from pulp.plugins.profiler import Profiler, InvalidUnitsRequested
-from pulp.server.agent import Context, PulpAgent
```

Comment:
I agree that this kind of change is ok, but I don't think it should be in a PEP-8 pull request. I think we want to minimize the possibility that these changes have unwanted side effects.